src/components/stop-all/stop-all.css

```diff
@@ -14,7 +14,10 @@
 }
 
 .stop-all:hover {
-    transform: scale(1.2);
+    /* Scale stop image by 1.2, but keep background static */
+    width: 1.5rem;
+    height: 1.5rem;
+    padding: 0.25rem;
```

Comment:
It looks like we're fighting with something here? AFAIK transform should help with issues like that since it doesn't affect width/height calculation.

Reply:
I was seeing an interesting rendering artifact due to layering when I moused over the stop button - a large white rectangle would briefly flash over the top left quarter of the stage. It didn't happen on hover for the green flag, so I switched the stop button to use the same css as the green flag. In any case, it seems like having consistent styling for the hover on the buttons makes sense.